Source/WebCore/rendering/RenderTable.cpp

```diff
@@ -1096,6 +1096,15 @@ LayoutUnit RenderTable::offsetHeightForColumn(const RenderTableCol& column) cons
     return m_columnOffsetHeight;
 }
 
+LayoutRect RenderTable::frameRectForColumn(const RenderTableCol& column) const
```

Comment:
I think frameRect is a slightly vague name for this geometry but then again what this function returns does not really have a spec definition (size is "border box", position is certainly not). It also seems very specific to background painting, so maybe it does not even have to be on the RenderTable and instead just have a local static function or a lambda instead.

Reply:
The name comes from RenderBox::frameRect(). References to frameRect() are very common in the Render classes. I put the function here by analogy with offsetLeftForColumn, etc.

Reply:
Indeed but it is a legacy term and new code should help us converge to a  more spec friendly codebase (and also "frame position" here seems to mean (slightly) different than what  RenderBox has) -but again, this does not necessarily have to show up at RenderTable.

Reply:
OK. The newer version of this PR does not have this code at all.